spec/features/crh_spec.rb

```diff
@@ -14,6 +14,12 @@
         expect(page).to_not have_content('We\'re sorry, we can\'t find the page you\'re looking for. It might have been removed, changed names, or is otherwise unavailable.')
       end
     end
+    describe 'crh show page works' do
```

Comment:
we are adding lots of functionality in this PR can we add tests to cover these scenarios...
some suggestions on tests to add:

- in the `va_facility_spec.rb` adding tests to ensure the CRHs are being displayed correctly

- in this `crh_spec.rb` adding tests to ensure the content displays correctly, tooltip works (needs to be added), search and filtering sorting works as intended

- in the `breadcrumbs_spec.rb` ensuring the breadcrumbs appear correctly and has the appropriate link for this new page

Reply:
Added tests for these cases..